doc/user/sql/show-sources.md

```diff
@@ -6,30 +6,48 @@ menu:
     parent: 'sql'
 ---
 
-`SHOW SOURCES` returns a list of all sources available to your Materialize instances.
+`SHOW SOURCES` returns a list of all sources available to your Materialize
+instances.
 
 ## Syntax
 
 {{< diagram "show-sources.html" >}}
 
+Field | Use
+------|-----
+_schema&lowbar;name_ | The schema to show sources from. Defaults to `public` in the current database. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
+
 ## Details
 
-Materialize maintains a number of views on your behalf, all of which begin with the name
-`mz_catalog.mz_`, e.g. `mz_catalog.mz_dataflow_channels`.
+### Internal statistic sources
+
+Materialize comes with a number of sources that contain internal statistics
+about the instance's behavior. These are kept in a "hidden" schema called
```

Comment:
The schema is just called `mz_catalog`, actually. It exists, magically, in every database.